src/components/nodes-table/nodes-table.js

```diff
@@ -0,0 +1,117 @@
+'esversion: 6'
```

Comment:
What is this for? Is it needed?

Reply:
I already removed this, not yet pushed.
It seems that there is some misconfiguration between VSCode and the npm build.
VSCode marks a lot of code to be invalid unless I add this directive - but it didn't help anyway.
Is there something I can do to configure VSCode to use the correct JavaScript language settings and code formatting? I didn't find anything in the README.

Reply:
Did you try to change `"editor.defaultFormatter": ` in .vscode/settings

Reply:
It's currently set to this:
`"editor.defaultFormatter": "numso.prettier-standard-vscode",`
What would be the correct value for Zwave2Mqtt?

Reply:
@ahochsteger That should be the correct one but try with another if it shows errros

Reply:
Ok, I found the setting in VSCode that caused many lines to be marked as warnings:
```JavaScript
"jshint.options": {
   "esversion": 6
}
```